### ISSUE TITLE
Latest/Recently Installed Apps (favourite like optional tab)

### ISSUE THREAD

**Issue by Bhardwaj10Ankit** [MM2-0/Kvaesitso#775]
Can @MM2-0  we please get an option to have a favorite apps like tag for Latest Installed apps in order off latest installed (which can be enabled or dissabled by user in favoutites or tag setting) so that we can see the apps recently installed and dont have to search or remember.

Usability : there can be many like, when we install multiple apps, it can be easy to find and etc..

A lot of other launchers also have these with a recent apps mark in most, so i'll suggest to please name it **Latest Apps** 😉 just so it stays unique as kvaesitso itself.

also @Sir-Photch have a look

**Comment by Bhardwaj10Ankit** [comment]
@MM2-0 please have a look at this, Can really be useful. 🙏🏻

**Comment by KOT040188** [comment]
I support adding this option. I can't imagine how you can manage apps without sorting them this way, especially when you have a lot of apps.

**Comment by Bhardwaj10Ankit** [comment]
@MM2-0 & @Sir-Photch have a look, give it a go if possible

**Comment by aleclarson** [comment]
@Bhardwaj10Ankit tagging people like that is rude in many cultures.

**Comment by getelman** [comment]
Good idea

**Comment by tasmo** [comment]
The only major feature I miss so far in Kvaesitso.

**Comment by cr1za** [comment]
I support this as well, please make a _recently installed_ filter for the search!

**Comment by hunterwilhelm** [comment]
I can work on this feature! It would have the following:

* Recently installed would show up to the right of the favorites
* You can toggle it in the settings


The only question is: Should be enabled by default? 

I'll start working on it disabled by default for now unless otherwise directed

**Comment by hunterwilhelm** [comment]
Before I make a PR, what do you think? 

(I put it in the settings only because the favorites button is for a different area than the apps list/grid, so a setting seemed great by itself)
 
https://github.com/user-attachments/assets/78c01047-c189-4b18-b22f-5dbc94e1932c

**Comment by Bhardwaj10Ankit** [comment]
@hunterwilhelm Instead of keeping a default sort option as recently installed, i would like the option to see receantly installed apps for a instance, as we do with the tags and all (kind of similar as it was in nova launcher). It serves a temporary usecase like, if we have just installed few aps, we can simply tap that option/section and see them sorted as their installed time

**Comment by hunterwilhelm** [comment]
@Bhardwaj10Ankit That makes sense, and should we sort all apps or just the 30 most recent? (Perhaps the count shown is configurable)

**Comment by Bhardwaj10Ankit** [comment]
@hunterwilhelm I think that if you're making a way/option to sort then all, and if you're adding a pannel like favourite then only as much allowed in the favourite grid

**Comment by hunterwilhelm** [comment]
@Bhardwaj10Ankit Thoughts on this? I believe this matches exactly what you're looking for.

https://github.com/user-attachments/assets/45a2a8c7-a73c-4f5b-b2d2-87fbfa9a56eb

**Comment by Bhardwaj10Ankit** [comment]
@hunterwilhelm let's go man. Finally 😁

**Comment by Bhardwaj10Ankit** [comment]
@hunterwilhelm also consider this as [frequents](https://github.com/MM2-0/Kvaesitso/issues/863)

**Comment by grantith** [comment]
Agreed, this feature would be a natural place to also include the option for frequently used apps.
